streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -111,36 +114,36 @@
     private final TaskId task2_2 = new TaskId(2, 2);
     private final TaskId task2_3 = new TaskId(2, 3);
 
-    private final Map<TaskId, Set<TopicPartition>> partitionsForTask = new HashMap<TaskId, Set<TopicPartition>>() {{
-            put(task0_0, Utils.mkSet(t1p0, t2p0));
-            put(task0_1, Utils.mkSet(t1p1, t2p1));
-            put(task0_2, Utils.mkSet(t1p2, t2p2));
-            put(task0_3, Utils.mkSet(t1p3, t2p3));
-
-            put(task1_0, Utils.mkSet(t3p0));
-            put(task1_1, Utils.mkSet(t3p1));
-            put(task1_2, Utils.mkSet(t3p2));
-            put(task1_3, Utils.mkSet(t3p3));
-
-            put(task2_0, Utils.mkSet(t4p0));
-            put(task2_1, Utils.mkSet(t4p1));
-            put(task2_2, Utils.mkSet(t4p2));
-            put(task2_3, Utils.mkSet(t4p3));
-        }};
+    private final Map<TaskId, Set<TopicPartition>> partitionsForTask = mkMap(
```

Comment:
"Double-brace initialization" is an anti-pattern. Switching usages to the appropriate utility methods.